tests/test_army_pay_decree_1503.py

```
@@ -809,3 +762,451 @@ def test_extractor_echo_with_dossier_origin_still_single_writer_after_close(game
+    monkeypatch.setattr(cb, "classify_cli_action_intent", fake_classify)
+    # 后置串行抽取器不得因前缀复活
+    for name in (
+        "extract_minister_actions",
+        "extract_draft_intent",
+        "extract_appointment_action",
+        "extract_confirmation_intent",
+    ):
+        monkeypatch.setattr(
+            cb, name,
+            lambda *a, **k: (_ for _ in ()).throw(
+                AssertionError(f"must not call {name} on explicit draft prefix")
```

Ruff (0.16.2)

[warning] Function definition does not bind loop variable `name`

(B023)